frontend/uddhar-frontend/src/coordinator/components/SearchOnMap.jsx

```
@@ -0,0 +1,104 @@
+  const [selectedLocation, setSelectedLocation] = useState(null);
+
+  const handleSearch = async (e) => {
+    e.preventDefault();
+    if (!searchQuery) return;
+    const response = await fetch(
+      `https://nominatim.openstreetmap.org/search?format=json&q=${searchQuery}`
+    );
+    const data = await response.json();
+    if (data.length > 0) {
+      setSearchQuery(data[0].display_name);
+      setSelectedLocation({ lat: parseFloat(data[0].lat), lon: parseFloat(data[0].lon), display_name: data[0].display_name });
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)